translate.py

```diff
@@ -1,11 +1,24 @@
 import re
 from pathlib import Path
-
+import numpy as np
 import bs4
 
 
 def translator(string: str) -> str:
-    return string.rstrip('.') + ')'
+    t = np.random.randint(1, 43)
+    pos = string.find('%1$@')
+    if (pos != -1):
```

Comment:
Meaningless brackets

Reply:
explain in more details please

Reply:
use `if pos != -1:` instead of `if (pos != -1):`

Reply:
not resolved